### DISCUSSION TITLE
"Void" category is not being ignored

### DISCUSSION THREAD

**Discussion by kadattack** [facebookresearch/detectron2#2834] [Q&A]
I'm using my own COCO dataset with the "ignore" category using ID number 4.
What is the proper way to set the "void" or "ignore" category so that model will ignore it while learning?

I've tried setting the Metadata
ignore_label=4
as shown here:
https://detectron2.readthedocs.io/en/latest/_modules/detectron2/data/datasets/coco_panoptic.html?highlight=ignore_label

And i've also tried setting the config file
SEM_SEG_HEAD.IGNORE_VALUE : 4
as shown here:
https://detectron2.readthedocs.io/en/latest/modules/config.html?highlight=ignore_value

None of these seem to work for me.
The model still learners the "ignore" value when visualising
Example of blue color ignore:
[https://i.imgur.com/e6iA4lN.jpeg](https://i.imgur.com/e6iA4lN.jpeg)

**Comment by junxiant** [comment]
Hey, did you manage to get an answer for this?
